README.md

```diff
@@ -9,53 +9,51 @@
 [![golangci-lint](https://github.com/Edgenesis/shifu/actions/workflows/golangci-lint.yml/badge.svg)](https://github.com/Edgenesis/shifu/actions/workflows/golangci-lint.yml)
 
 </div>
-
+ 
 <div align="center">
 
-<img width="200px" src="./img/shifu-logo.svg"></img>
-
-
-Shifu is a Kubernetes-native IoT development framework that 
-
-greatly improves the efficiency, quality and reusability of IoT application development.
-
+<img width="300px" src="./img/shifu-logo.svg"></img>
+<div align="center">
 
-|Feature|Description |
-|---|---|
-|🔌 Fast Device integration &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|Compatible with almost all protocols and drivers.|
-|👨‍💻 Efficient Application development|Shifu structually virtualizes each device and expose its capabilities in the form of APIs.|
-|👨‍🔧 Easy Operation & Maintenance|Kubernetes-native framework, sparing the need for maintaining an additional O&M infrastructure.|
+<h1 style="border-bottom: none">
+<br>
+    Open Source Alternative to GCP IoT Core
+    <br />
+</h1>
+Shifu is a Kubernetes native, production-grade, protocol & vendor agnostic IoT development framework.
+</div>
 </div>
 <br/><br/>
 
-# 🪄 Demo
 <div align="center">
-<img width="900px" src="./img/demo-camera.gif"></img>
-<img width="900px" src="./img/demo-plc.gif"></img>
+    <a href="https://discord.gg/2tbdBrkGHv"><img src="https://img.shields.io/badge/-Discord-424549?style=social&logo=discord" height=25></a>
+    &nbsp;
+    <a href="https://twitter.com/ShifuFramework"><img src="https://img.shields.io/badge/-Twitter-red?style=social&logo=twitter" height=25></a>
+    &nbsp;
+    <a href="https://www.linkedin.com/company/76257633/admin/"><img src="https://img.shields.io/badge/-Linkedin-red?style=social&logo=linkedin" height=25></a>
+     &nbsp;
+    <a href="https://github.com/Edgenesis/shifu"><img src="https://img.shields.io/github/stars/Edgenesis/shifu?style=social" height=25></a>
 </div>
-<br/><br/>
 
-# 🔧 Install
+# Features
+**Kubernetes native** — developing your application while managing your devices, spare the need for maintaining an additional O&M infrastructure
 
-- If you have a running Kubernetes cluster: Please use the command `kubectl apply` to install Shifu in your cluster:
+**Open Platform**— avoide vendor lock-in, you can easily deploy Shifu on public, private and hybrid cloud. 
```

Comment:
**Open Platform**— No vendor lock-in. You can easily deploy Shifu on the edge(from RaspberryPi to edge clusters) or on the cloud(public, private and hybrid cloud are all supported).